src/dtx/dtx_common.c

```diff
@@ -700,7 +700,6 @@ dtx_batched_commit(void *arg)
 		    dtx_hlc_age2sec(stat.dtx_oldest_active_time) >=
 		    DTX_CLEANUP_THD_AGE_UP) {
 			D_ASSERT(!dbca->dbca_cleanup_done);
-			sleep_time = 0;
```

Comment:
is this intended?

Reply:
Yes, we do not want to DTX cleanup ULT to be scheduled too frequently.